webui/ec2/EC2.py

```diff
@@ -2,6 +2,21 @@
 from .models import User, AccessKey
 import boto3
 from botocore.exceptions import ClientError
+from django.db import transaction
+from django.utils import timezone
+from ec2.models import Instance
+import json
+
+
+regions = [
```

Comment:
can't you get this list at runtime from Amazon ? Suspecting that this list static more or less so not a big deal actually but still ...

Reply:
I will give it a try.